### ISSUE TITLE
cursorline in popup window makes the signcolumn show up

### ISSUE THREAD

**Issue by Yggdroot** [vim/vim#5213]
**Describe the bug**
cursorline in popup window makes the signcolumn show up

**To Reproduce**

test.vim
```vim
call popup_create(bufnr('%'), #{cursorline: 1})
"call popup_create(bufnr('%'), #{cursorline: 1})
"call popup_create(bufnr('%'), #{cursorline: 1})
"call popup_create(bufnr('%'), #{cursorline: 1})
"call popup_create(bufnr('%'), #{cursorline: 1})
"call popup_create(bufnr('%'), #{cursorline: 1})
```
1.  `echo &signcolumn`  outputs 'auto', no signcolumn
2. :so %
2. signcolumn show up

**Expected behavior**
no signcolumn

**Environment (please complete the following information):**
 - Vim version : 8.1.2267
 - OS:  CentOS 7
 - Terminal: GNOME Terminal

**Comment by bfrg** [comment]
I just stumbled upon this issue. I think it was solved in 4eb7dae.

**Comment by Yggdroot** [comment]
Confirmed, it's fixed. Thanks.
